addons/l10n_it_edi/models/l10n_it_edi_import.py

```diff
@@ -0,0 +1,346 @@
+# -*- coding:utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import re
+from datetime import datetime
+import logging
+
+from odoo import models, fields, _
+
+_logger = logging.getLogger(__name__)
+
+
+def gettext(tree, xpath, many=False):
```

Comment:
why not define them at least in the class and as get_text instead?

Reply:
They're not in the class to avoid calling it on an instance of an object
A couple of those are actually imported in another class.
Maybe I should put those in a separate file, like `tools/utils.py` or sth?
For the underscore: 👍🏻  
![image](https://github.com/odoo/odoo/assets/1665365/aacf49e2-1311-48bf-a567-dd80d988e73e)